### PR TITLE
Add tests for ansible.utils

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gosriniv@redhat.com>

Depends-On: https://github.com/ansible/project-config/pull/647

### CODE REVIEW

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -1323,6 +1323,132 @@
     vars:
       ansible_collections_repo: github.com/ansible-collections/ansible.posix
 
+- job:
+    name: ansible-test-network-integration-base-ansible-utils
```
Comment (pabelanger) [inline zuul.d/jobs.yaml:1327]: we can drop all of this, we don't have integration jobs right now

File: zuul.d/project-templates.yaml [hunk 2]
```diff
@@ -99,6 +99,36 @@
...
+        - ansible-test-units-ansible-utils-python27
+        - ansible-test-units-ansible-utils-python35
+        - ansible-test-units-ansible-utils-python36
+        - ansible-test-units-ansible-utils-python37
+        - ansible-test-units-ansible-utils-python38
+    gate:
+      queue: integrated
+      jobs: *ansile-collections-ansible-utils-units-jobs
+
+- project-template:
```
Comment (pabelanger) [inline zuul.d/project-templates.yaml:118]: we can delete this for now

File: zuul.d/jobs.yaml [hunk 3]
```diff
@@ -1323,6 +1323,132 @@
...
+    post-run:
+      - playbooks/ansible-test-network-integration-base/post.yaml
+      - playbooks/ansible-test-base/post.yaml
+    required-projects:
+      - name: github.com/ansible/ansible
+      - name: github.com/ansible-collections/ansible.utils
+    timeout: 10800
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/ansible.utils
+      ansible_test_command: network-integration
```
Comment (ganeshrn) [inline zuul.d/jobs.yaml:1345]: We don't need network-integration test to be enabled for `ansible.utils` collection. The command I use to run the integration test for `ansible.utils` locally is for eg: `ansible-test integration -vvvv --python 3.8`

File: zuul.d/project-templates.yaml [hunk 4]
```diff
@@ -99,6 +99,22 @@
       queue: integrated
       jobs: *ansible-collections-arista-eos-jobs
 
+- project-template:
+    name: ansible-collections-ansible-utils-units
+    check:
+      jobs: &ansile-collections-ansible-utils-units-jobs
+        - build-ansible-collection
```
Comment (pabelanger) [inline zuul.d/project-templates.yaml:106]: this is the wrong location, it should be in ansible-collections-ansible-utils project-template

Comment (pabelanger) [reply zuul.d/project-templates.yaml:106]: but we can move it later

**Pull request merged**